docs/index.md

```diff
@@ -1,21 +1,16 @@
-# MONAI Deploy Informatics Gateway <small>v0.0.0</small>
```

Comment:
we will need `v0.0.0` for the build agent to automagically stamp the version number.